campaign-launcher/interfaceV2/src/utils/TokenManager.ts

```diff
@@ -0,0 +1,65 @@
+import { decodeJwt } from '.';
+import { ACCESS_TOKEN_KEY, REFRESH_TOKEN_KEY } from '../constants';
+
+export type TokenData = {
+  access_token: string;
+  refresh_token: string;
+};
+
+export class TokenManager {
+  private static instance: TokenManager;
+
+  private constructor() {}
+
+  static getInstance(): TokenManager {
+    if (!TokenManager.instance) {
+      TokenManager.instance = new TokenManager();
+    }
+    return TokenManager.instance;
+  }
+
+  getAccessToken(): string | null {
+    return localStorage.getItem(ACCESS_TOKEN_KEY);
+  }
+
+  getRefreshToken(): string | null {
+    return localStorage.getItem(REFRESH_TOKEN_KEY);
+  }
+
+  setTokens(tokens: TokenData): void {
+    localStorage.setItem(ACCESS_TOKEN_KEY, tokens.access_token);
+    localStorage.setItem(REFRESH_TOKEN_KEY, tokens.refresh_token);
+  }
+
+  setAccessToken(token: string): void {
+    localStorage.setItem(ACCESS_TOKEN_KEY, token);
+  }
+
+  setRefreshToken(token: string): void {
+    localStorage.setItem(REFRESH_TOKEN_KEY, token);
+  }
```

Comment:
Probably we don't need these two: tokens always go in pairs

Reply:
i would leave these, let it be